src/Nevermind/Nevermind.Core/Encoding/TransactionDecoder.cs

```diff
@@ -60,6 +60,7 @@ internal Transaction Decode(object[] data)
                 transaction.Signature = signature;
             }
 
+            transaction.RecomputeHash();
```

Comment:
any suggestion how to do that nicer? I did not like it in my code (for block header) and wanted to refactor

Reply:
maybe some change tracking and getter for hash that would always recompute when onvoked and any changes were made...